wof/__init__.py

```diff
@@ -7,3 +7,10 @@
 
 from wof.core_1_0 import WOF
 from wof.core_1_1 import WOF_1_1
+
+__all__ = [
```

Comment:
@ocefpaf I forgot what `__all__` does? These stuff in all seem to be imported already? Thanks